search-sentiments-map/routes/search.js

```diff
@@ -65,15 +67,22 @@ router.get('/:topic/:countries', (req, res) => {
 });
 
 /** 
- * Returns a JSON-formatted array of search results for all countries retrieved
- * from the Datastore.
+ * Returns a JSON-formatted array of search results from specified time range
+ * for all countries retrieved from the Datastore.
  * @param {string} topic Search topic to get data for.
+ * @param {number} timeRange An integer representing how many 
+ *     `CURRENT_DATA_TIME_RANGE_12_HOURS_MS` ranges previous to get data from.
```

Comment:
Explain in words. In general we want to explain things in English and use code only when required.

Reply:
```suggestion
 * @param {number} timeRange An integer representing how many 
 *    12 hour time ranges previous to get data from.
```

So I should do this instead? I am a little confused because in this previous pull request which was a similar situation you told me to use the constant and not the time.

https://github.com/googleinterns/step54-2020/pull/13#discussion_r456738898